workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -1026,16 +1026,39 @@ def write_tsv_json(job_context):
         return [tsv_path]
 
 
-def download_computed_file(download_tuple: Tuple[ComputedFile, str]):
-    """ this function downloads the latest computed file. Receives a tuple with
-    the computed file and the path where it needs to be downloaded
-    This is used to parallelize downloading quantsf files. """
-    (latest_computed_file, output_file_path) = download_tuple
+def download_quant_file(download_tuple: Tuple[Sample, ComputedFile, str]) -> str:
+    """ this function downloads the latest computed file and then returns the
+    failure reason as a string, if there was one. Receives a tuple with the
+    computed file and the path where it needs to be downloaded This is used to
+    parallelize downloading quantsf files. """
+    (sample, latest_computed_file, output_file_path) = download_tuple
     try:
         latest_computed_file.get_synced_file_path(path=output_file_path)
-    except:
+    except Exception as e:
         # Let's not fail if there's an error syncing one of the quant.sf files
-        logger.exception("Failed to sync computed file", computed_file_id=latest_computed_file.pk)
+        logger.exception(
+            "Failed to sync computed file", computed_file_id=latest_computed_file.pk, exception=e
```

Comment:
`exception=e` isn't necessary here as `logger.exception` will use the current stack traceback info to log the stacktrace and the exception.

Reply:
Oh sweet